### DISCUSSION TITLE
Pi Karaoke working on Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by BentWrench** [vicwomg/pikaraoke#311] [General]
I bought a Pi 5 and loaded Pi Karaoke on it tonight, and it is running just fine. Only minor issue, the CPU shows 0.0% use and the Pikaraoke version shows version 1.1.2, even though this is a fresh git clone install from today, and I think it should be 1.2. Oh and I couldn't connect using hostname so switched to using --prefer-ip when launching. Other than that, runs pretty snappy on a Pi 5. Thanks Vic!!!

Edit: CPU usage showed 5% at one point. But now back to 0.0%. Probably works OK depending on when you look at it.

**Comment by vicwomg** [comment]
Thank you for pointing out the version mismatch, that's an issue with the constants.py file. I will fix on master

**Comment by Jake-pi** [comment]
I did the same, and song downloads are much much faster now. Because the Pi5 8gb was kind of expensive for just a karaoke player, I installed the latest release of Bullseye with desktop and all recommendations options. Then I used SSH to follow the PiKaraoke installation instructions, then made a desktop icon to launch PiKaraoke. I can use my new Pi5 as a computer on my TV, and launch PiKaraoke from the desktop. The Pi5 now acts as a media centre with internet, and PiKaraoke.
